app/(tabs)/index.tsx

```
@@ -28,6 +53,67 @@ const HomeScreen = () => {
   const notificationColor = theme === 'dark' ? '#FFFFFF' : '#000000';
 
 
+  // Initialize Health Connect and fetch data
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on initializeHealthTracking.

This dependency is being used here, but is not specified in the hook dependency list.

Either include it or remove the dependency array.
Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)